pkg/database/user.go

```diff
@@ -43,6 +43,9 @@ type User struct {
 	Name        string `gorm:"type:varchar(100)"`
 	SystemAdmin bool   `gorm:"column:system_admin; default:false;"`
 
+	RememberLastUsedSMSTemplate bool   `gorm:"type:boolean; not null; default:true;"`
+	DefaultSMSTemplateLabel     string `gorm:"type:varchar(255);"`
```

Comment:
A few things here:

- A user can be a member of multiple realms (with different templates), so this might be better as a property on the `Membership` instead of the user
- There needs to be some validation that the template is one of the keys in the hstore for the realm
- There needs to be some validation/changes when the hstore column is updated to cascade here

Reply:
I don't think we need to keep the membership preferred template in sync with the existing ones. When the page loads and the template doesn't exist, that selectedIf doesn't match anything and the user goes back to the default template in the selector.

Reply:
how does this work for users that in in more than 1 realm?

Reply:
This uses Seth's suggestion now - it's attached to Membership

Reply:
so I have to join a club?

Reply:
Club Realm

Reply:
™️